packages/sirius-web/frontend/sirius-web-application/src/views/library-browser/useLibraries.types.ts

```diff
@@ -11,6 +11,8 @@
  *     Obeo - initial API and implementation
  *******************************************************************************/
 
+import { GQLEditingContext } from '../edit-project/useProjectAndRepresentationMetadata.types';
```

Comment:
Don't reuse types like that, define your own, you don't have the same GraphQL query

Reply:
Done